index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Nahan T. Baker: Nashville Developer</title>
+    <meta charset="UTF-8">
+    <meta name="description" content="I'm a Franklin-based developer available for full-time work in April 2017 after I graduate from Nashville Software School.">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">  
```

Comment:
Be very careful about leaving trailing whitespace in your code. It will inevitably lead to merge conflicts. You can see whitespace by adding a plugin called `Trailing Spaces` to your sublime, or you can add this line to your `User Preferences` and stop them out right. `"trim_trailing_white_space_on_save": true`